scripts/2011/09/generate_deworks.py

```diff
@@ -69,45 +66,45 @@ def get_ia_metadata(self, identifier):
         else:
             # the requested identifier is not found.
             # returning a fake metadata
-            return {"collection":[]}
+            return {"collection": []}
 
     def close(self):
         """Removes all the temp files created for running map-reduce.
         """
         shutil.rmtree(self.tmpdir, ignore_errors=True)
 
     def map(self, key, value):
-        """Takes key and json as inputs and emits (work-key, work-json) or (work-key, edition-json)
-        for work and edition records respectively.
+        """Takes key and json as inputs and emits (work-key, work-json) or
+        (work-key, edition-json) for work and edition records respectively.
         """
         doc = simplejson.loads(value)
-        type_key = doc['type']['key']
+        type_key = doc["type"]["key"]
 
-        if type_key == '/type/edition':
```

Comment:
@cclauss https://github.com/internetarchive/openlibrary/issues/2981 fine that we're changing single quotes to double as per `black` linting. Can we please move these in a different PR to separate the style v. the content changes?